test/AutoRest.TestServer.Tests/TypeSchemaMappingTest.cs

```diff
@@ -100,7 +100,7 @@ public void ObjectsAreMappedToSchemas()
         [Test]
         public void NullableParamsAsNull()
         {
-            var model = new RenamedModelStruct("test", null, null);
```

Comment:
should we use an explicit parameter name here to test that we get the expected name?

Reply:
Maybe worth adding something to https://github.com/Azure/autorest.csharp/blob/feature/v3/test/AutoRest.TestServer.Tests/Common/StringExtensionsTests.cs